.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          GIT_DOMAIN_OPTION=""
+          if [[ -n "${{ vars.GIT_DOMAIN }}" ]]; then
+            GIT_DOMAIN_OPTION=" --git.domain=${{ vars.GIT_DOMAIN }}"
+          fi
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${GIT_DOMAIN_OPTION}" >> $GITHUB_ENV
```

Comment:
**Security Issue**: Potential exposure of sensitive tokens through environment variables in logs or error messages. <br> **Fix**: Ensure that sensitive tokens like GIT_ACCESS_TOKEN and BITO_ACCESS_KEY are not directly echoed or logged. Use secrets masking or environment variable references that do not expose the actual values. <br> **Code Suggestion**: 
 ```
 -          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${GIT_DOMAIN_OPTION}" >> $GITHUB_ENV
 +          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=*** --bito_cli.bito.access_key=***${GIT_DOMAIN_OPTION}" >> $GITHUB_ENV
 ```